<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Project_DM_Qt</class>
 <widget class="QMainWindow" name="Project_DM_Qt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1110</width>
    <height>800</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1110</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1110</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Project_DM_Qt</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame_Buttons">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>720</y>
      <width>1111</width>
      <height>61</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="Btn_natural">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
color: #000000;
background-color: white;
border: 1px solid #828282;
border-radius: 3px;
font-size: 18px;
}
QPushButton:hover {
border: 1px solid #828282;
background-color: #d5d5d5;
}
QPushButton:hover:pressed { 
background-color: gray;
color: white;
}
QPushButton:disabled { 
background-color: gray;
color: white;
}</string>
       </property>
       <property name="text">
        <string>Натуральные числа N</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Btn_integer">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
color: #000000;
background-color: white;
border: 1px solid #828282;
border-radius: 3px;
font-size: 18px;
}
QPushButton:hover {
border: 1px solid #828282;
background-color: #d5d5d5;
}
QPushButton:hover:pressed { 
background-color: gray;
color: white;
}
QPushButton:disabled { 
background-color: gray;
color: white;
}</string>
       </property>
       <property name="text">
        <string>Целые числа Z</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Btn_rational">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
color: #000000;
background-color: white;
border: 1px solid #828282;
border-radius: 3px;
font-size: 18px;
}
QPushButton:hover {
border: 1px solid #828282;
background-color: #d5d5d5;
}
QPushButton:hover:pressed { 
background-color: gray;
color: white;
}
QPushButton:disabled { 
background-color: gray;
color: white;
}</string>
       </property>
       <property name="text">
        <string>Рациональные числа Q</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Btn_polynomials">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
color: #000000;
background-color: white;
border: 1px solid #828282;
border-radius: 3px;
font-size: 18px;
}
QPushButton:hover {
border: 1px solid #828282;
background-color: #d5d5d5;
}
QPushButton:hover:pressed { 
background-color: gray;
color: white;
}
QPushButton:disabled { 
background-color: gray;
color: white;
}</string>
       </property>
       <property name="text">
        <string>Многочлены P</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Btn_matrix">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
color: #000000;
background-color: white;
border: 1px solid #828282;
border-radius: 3px;
font-size: 18px;
}
QPushButton:hover {
border: 1px solid #828282;
background-color: #d5d5d5;
}
QPushButton:hover:pressed { 
background-color: gray;
color: white;
}
QPushButton:disabled { 
background-color: gray;
color: white;
}</string>
       </property>
       <property name="text">
        <string>Матрицы M</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="main" native="true">
    <property name="geometry">
     <rect>
      <x>-1110</x>
      <y>0</y>
      <width>5550</width>
      <height>721</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>5550</width>
      <height>721</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>5550</width>
      <height>721</height>
     </size>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <property name="spacing">
      <number>0</number>
     </property>
     <item row="0" column="2">
      <widget class="QWidget" name="rational" native="true"/>
     </item>
     <item row="0" column="3">
      <widget class="QWidget" name="polynomials" native="true"/>
     </item>
     <item row="0" column="1">
      <widget class="QWidget" name="integer" native="true">
       <widget class="QWidget" name="widget" native="true">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>220</y>
          <width>491</width>
          <height>281</height>
         </rect>
        </property>
        <widget class="QTextEdit" name="Integer_num1">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>100</y>
           <width>91</width>
           <height>41</height>
          </rect>
         </property>
        </widget>
        <widget class="QComboBox" name="Integer_choose">
         <property name="geometry">
          <rect>
           <x>150</x>
           <y>110</y>
           <width>40</width>
           <height>22</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">font-size: 18px;</string>
         </property>
         <item>
          <property name="text">
           <string>+</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>-</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>*</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>/</string>
          </property>
         </item>
        </widget>
        <widget class="QTextEdit" name="Integer_num2">
         <property name="geometry">
          <rect>
           <x>200</x>
           <y>100</y>
           <width>91</width>
           <height>41</height>
          </rect>
         </property>
        </widget>
        <widget class="QTextBrowser" name="Integer_res">
         <property name="geometry">
          <rect>
           <x>320</x>
           <y>100</y>
           <width>91</width>
           <height>41</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="Integer_Error">
         <property name="geometry">
          <rect>
           <x>300</x>
           <y>99</y>
           <width>21</width>
           <height>41</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">font-size: 18px;</string>
         </property>
         <property name="text">
          <string>=</string>
         </property>
        </widget>
        <widget class="QPushButton" name="Integer_Button">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>170</y>
           <width>201</width>
           <height>51</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
color: #000000;
background-color: white;
border: 1px solid #828282;
border-radius: 3px;
font-size: 18px;
}
QPushButton:hover {
border: 1px solid #828282;
background-color: #d5d5d5;
}
QPushButton:hover:pressed { 
background-color: gray;
color: white;
}
QPushButton:disabled { 
background-color: gray;
color: white;
}</string>
         </property>
         <property name="text">
          <string>Вычислить</string>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QWidget" name="natural" native="true"/>
     </item>
     <item row="0" column="4">
      <widget class="QWidget" name="matrix" native="true">
       <widget class="QLabel" name="m_pic5">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>120</y>
          <width>47</width>
          <height>241</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>../../../Desktop/Skobka.jpg</pixmap>
        </property>
       </widget>
       <widget class="QLabel" name="m_pic3">
        <property name="geometry">
         <rect>
          <x>370</x>
          <y>380</y>
          <width>47</width>
          <height>241</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>../../../Desktop/Skobka.jpg</pixmap>
        </property>
       </widget>
       <widget class="QLabel" name="m_pic4">
        <property name="geometry">
         <rect>
          <x>720</x>
          <y>380</y>
          <width>47</width>
          <height>241</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>../../../Desktop/Skobka1.jpg</pixmap>
        </property>
       </widget>
       <widget class="QSpinBox" name="spinBox">
        <property name="geometry">
         <rect>
          <x>520</x>
          <y>140</y>
          <width>71</width>
          <height>31</height>
         </rect>
        </property>
        <property name="toolTip">
         <string>Matrix size</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="minimum">
         <number>2</number>
        </property>
        <property name="maximum">
         <number>32767</number>
        </property>
        <property name="value">
         <number>3</number>
        </property>
       </widget>
       <widget class="QLabel" name="matrix1">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>110</y>
          <width>121</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Bookman Old Style</family>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Matrix 1</string>
        </property>
       </widget>
       <widget class="QLabel" name="m_Result">
        <property name="geometry">
         <rect>
          <x>510</x>
          <y>370</y>
          <width>91</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Bookman Old Style</family>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Result</string>
        </property>
       </widget>
       <widget class="QLabel" name="m_operation_label">
        <property name="geometry">
         <rect>
          <x>520</x>
          <y>200</y>
          <width>71</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>MS Shell Dlg 2</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string> Operation</string>
        </property>
       </widget>
       <widget class="QScrollArea" name="m_scrollArea_1">
        <property name="geometry">
         <rect>
          <x>620</x>
          <y>140</y>
          <width>331</width>
          <height>201</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::WinPanel</enum>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents_8">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>327</width>
           <height>197</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout_14">
          <item row="0" column="0">
           <layout class="QGridLayout" name="m_gridLayout_1"/>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QLabel" name="m_pic1">
        <property name="geometry">
         <rect>
          <x>600</x>
          <y>120</y>
          <width>47</width>
          <height>241</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>../../../Desktop/Skobka.jpg</pixmap>
        </property>
       </widget>
       <widget class="QScrollArea" name="m_scrollArea">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>140</y>
          <width>331</width>
          <height>201</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::WinPanel</enum>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents_9">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>327</width>
           <height>197</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout_15">
          <item row="0" column="0">
           <layout class="QGridLayout" name="m_gridLayout"/>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QComboBox" name="m_operation">
        <property name="geometry">
         <rect>
          <x>520</x>
          <y>220</y>
          <width>69</width>
          <height>22</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="currentText">
         <string>+</string>
        </property>
        <property name="frame">
         <bool>true</bool>
        </property>
        <item>
         <property name="text">
          <string>+</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>*</string>
         </property>
        </item>
       </widget>
       <widget class="QLabel" name="m_pic2">
        <property name="geometry">
         <rect>
          <x>950</x>
          <y>120</y>
          <width>47</width>
          <height>241</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>../../../Desktop/Skobka1.jpg</pixmap>
        </property>
       </widget>
       <widget class="QPushButton" name="m_calc">
        <property name="geometry">
         <rect>
          <x>520</x>
          <y>290</y>
          <width>71</width>
          <height>51</height>
         </rect>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>Calc</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QLabel" name="matrix2">
        <property name="geometry">
         <rect>
          <x>730</x>
          <y>110</y>
          <width>121</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Bookman Old Style</family>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Matrix 2</string>
        </property>
       </widget>
       <widget class="QScrollArea" name="m_scrollArea_2">
        <property name="geometry">
         <rect>
          <x>390</x>
          <y>400</y>
          <width>331</width>
          <height>201</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents_10">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>327</width>
           <height>197</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout_17">
          <item row="0" column="0">
           <layout class="QGridLayout" name="m_gridLayout_2"/>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QLabel" name="matrix_size">
        <property name="geometry">
         <rect>
          <x>520</x>
          <y>120</y>
          <width>71</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Matrix size&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QLabel" name="m_pic6">
        <property name="geometry">
         <rect>
          <x>490</x>
          <y>120</y>
          <width>31</width>
          <height>241</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>../../../Desktop/Skobka1.jpg</pixmap>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1110</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Настройки</string>
    </property>
    <addaction name="action"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="action">
   <property name="text">
    <string>Тест</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
